shuup/front/migrations/0004_move_basket_model.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.12 on 2017-01-20 19:49
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import shuup.core.models._stored_basket
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('shuup_front', '0003_basket_key_unique'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='storedbasket',
+            name='key',
+            field=models.CharField(db_index=True, default=shuup.core.models._stored_basket.generate_key, max_length=32, unique=True),
```

Comment:
Not how sure you managed to generate this under front, but the stored basket should have migration in core.